istio/datadog_checks/istio/istio.py

```diff
@@ -7,24 +7,20 @@
 from datadog_checks.base.checks.openmetrics import OpenMetricsBaseCheck
 from datadog_checks.base.errors import CheckException
 
+from .constants import CITADEL_NAMESPACE, GALLEY_NAMESPACE, MESH_NAMESPACE, MIXER_NAMESPACE, PILOT_NAMESPACE
+from .metrics import CITADEL_METRICS, GALLEY_METRICS, GENERIC_METRICS, MESH_METRICS, MIXER_METRICS, PILOT_METRICS
 
-class Istio(OpenMetricsBaseCheck):
-    MIXER_NAMESPACE = 'istio.mixer'
-    MESH_NAMESPACE = 'istio.mesh'
-    PILOT_NAMESPACE = 'istio.pilot'
-    GALLEY_NAMESPACE = 'istio.galley'
-    CITADEL_NAMESPACE = 'istio.citadel'
-    DEFAULT_METRIC_LIMIT = 0
```

Comment:
Need this

Reply:
These were moved to `constants.py`

Reply:
Yes but it has no effect on anything when not bound to the class

Reply:
- https://github.com/DataDog/integrations-core/blob/abce9342b07e38acb68f7c060f524281add9faf6/datadog_checks_base/datadog_checks/base/checks/base.py#L274
- https://github.com/DataDog/integrations-core/blob/abce9342b07e38acb68f7c060f524281add9faf6/datadog_checks_base/datadog_checks/base/checks/openmetrics/base_check.py#L35